frontend/src/admin/new-survey-button.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {FunctionComponent, useRef} from "react"
+import {ztorFetch} from "../services/ztor-fetch"
+import {Project} from "zero-zummon"
+import {Menu} from "primereact/menu"
+import {Button} from "primereact/button"
+import {MenuItem} from "primereact/menuitem"
+import {useNavigate} from "react-router-dom"
+import {usePromise} from "../hooks/use-promise"
+import {assertDefined} from "../services/util"
+
+export const NewSurveyButton: FunctionComponent = () => {
```

Comment:
a more generic name that explains what it does. Like projects-menu

Reply:
It's a button which lets you fill out a new survey :wink: